vtex-apps/order-quote
# [ORDERQUOTE-16] Change discounts, shipping, and taxes data type to Float

#### What does this PR do? \*
Update the data type for `discount`, `shipping`, and `taxes` in the MasterData schema from `integer` to `number` and from `Int` to `Float` in the GraphQL schema. Change made so GraphQL can accept larger numbers for those fields.

#### How to test it? \*
Test in this [workspace](https://supp--coexitob2b.myvtex.com/). See [ticket](https://vtex-dev.atlassian.net/browse/ORDERQUOTE-16) for login credentials. 

1. Login using credentials from ticket
2. Add SKUs [175](https://supp--coexitob2b.myvtex.com/bateria-mac-silver-31t1250mc/p) and [248](https://supp--coexitob2b.myvtex.com/bateria-mac-silver-8d1600mc/p) to cart
3. Create new quote and save
